web_src/js/features/repo-home.js

```diff
@@ -1,32 +1,45 @@
 import $ from 'jquery';
 import {stripTags} from '../utils.js';
 import {hideElem, showElem} from '../utils/dom.js';
+import {htmlEscape} from 'escape-goat';
+import {svg} from '../svg.js';
 
 const {appSubUrl, csrfToken} = window.config;
 
 export function initRepoTopicBar() {
   const mgrBtn = $('#manage_topic');
-  const editDiv = $('#topic_edit');
-  const viewDiv = $('#repo-topics');
+  if (!mgrBtn.length) return;
+
   const saveBtn = $('#save_topic');
+  const topicListDiv = $('#repo-topics');
   const topicDropdown = $('#topic_edit .dropdown');
-  const topicForm = $('#topic_edit.ui.form');
-  const topicPrompts = getPrompts();
+  const topicDropdownSearch = topicDropdown.find('input.search');
+  const topicForm = $('#topic_edit');
+  const topicPrompts = {
+    countPrompt: topicDropdown.attr('data-text-count-prompt'),
+    formatPrompt: topicDropdown.attr('data-text-format-prompt'),
+    remove: topicDropdown.attr('data-text-remove'),
+  };
+
+  function addLabelDeleteIconAria($el) {
+    $el.removeAttr('aria-hidden').attr({
+      'aria-label': topicPrompts.remove,
```

Comment:
Is recommended to add the title of the item which is going to be deleted as a better label for each remove button. The same applies to typical links labeled as "See more..." in other contexts. Many automatic accessibility conformance tests warn repetitions of labels because are confuse to the end user; if you think, with each TAB pressing, a blind person doesn't know clearly to which context arrives, so "Remove" as a generic action has unclear consequences.

@delvh also suggested adding the "topic" word in my PR #22631, which is even better.

Reply:
Before I thought since the "delete icon"  is not really focusable,  so a common "Remove"   is enough.

If you prefer "Remove topic", we can add it.

ps: I can not 100% understand the meaning of "each TAB pressing", IMO due to the dropdown problem the other elements are not focusable. Could you explain more details about how to do that "each TAB pressing" operation?